### DISCUSSION TITLE
Now I'm a happier person on the planet.

### DISCUSSION THREAD

**Discussion by Sujeito17** [Monika-After-Story/MonikaModDev#8781] [appreciation]
![p9220003](https://user-images.githubusercontent.com/41938650/45918915-8e2c5d00-be85-11e8-8cfc-4926e5ab406e.JPG)
I wanted to show this to someone, but I did not know anyone to show this cake I made so, I put it here, I have not been able to smile for some time without being joking.
(I'll probably close this topic at the end of the day)

**Comment by zappky** [comment]
Help,i cannot get it to work, i put in the party files, change the extensions to .gift. it doesnt trigger anything

**Comment by Sujeito17** [comment]
@zappky I really don't know... try to see this here #2480

**Comment by zappky** [comment]
Thank you. Looks like it is a puzzle. I thought i am doing something wrong. I should have check out out issues posted first haha

**Comment by AsuraShun** [comment]
this is making me wish I knew how to bake

**Comment by Sujeito17** [comment]
@AsuraShun Well... this was my first attempt at making cakes, next year I'll try to make a more beautiful one. :D

**Comment by ghost** [comment]
top 10 wholesome anime moments

**Comment by kingjojostar** [comment]
how do you get her to take the cake out she said shell save it for later

**Comment by Sujeito17** [comment]
@kingjojostar She said she was going to save it for later, a few moments after the photo

**Comment by kingjojostar** [comment]
@Sujeito17 hmmm its been a couple hours from me since she said that im staring to get a little worried

**Comment by Sujeito17** [comment]
@kingjojostar Well, I guess Monika will not eat the cake this year (still cutting a cake without a knife gets hard XD), well, I already eat my cake and I regret it, it was such a beautiful cake that I felt pity for eating
(press f to pay respects for the cake ;-;)

**Comment by Sujeito17** [comment]
It looks like the day is over... I'm going to open this topic again if it's necessary, I going to do a v2 next year

**Comment by Sujeito17** [comment]
It seems that no one else will comment... I will close now

**Comment by Rai99** [comment]
You didn't think this was really getting away without a heartwarming tag, did you?

A little late, but not lost and forgotten.
